libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1446,6 +1446,32 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 		_functionCall.annotation().type = make_shared<TupleType>(functionType->returnParameterTypes());
 
 	TypePointers parameterTypes = functionType->parameterTypes();
+
+	if (
```

Comment:
Perhaps better to use `_functionType.padArguments() || _functionType.takesArbitraryParameters()`

Reply:
Updated.